doc/theory/glossary.tex

```diff
@@ -358,11 +516,34 @@
 	\Lambda_i(t) = \left[ \min \{t-b_i, d_i-t\}\right]_+	
 	\end{equation*}	
 	and $c_+ := \max(c,0)$. When $w_i = \vert d_i - b_i \vert^p$ for $0 < p \leq \infty$ we refer to $\phi$ as the \textit{$p$-power-weighted silhouette} of $D$. The silhouette construction defines a
-	%\hyperref[vectorization_kernel_and_amplitude]{vectorization}
-	vectorization of the set of persistence diagrams with target the vector space of continuous real-valued functions on $\mathbb R$.	
+
+	%\hyperref[vectorization_amplitude_and_kernel]{vectorization}
+	vectorization of the set of persistence diagrams with target the vector space of continuous real-valued functions on $\mathbb R$.
 	
 	\paragraph{\\ References:} \cite{chazal2014stochastic}
 	
+	\subsection*{Heat vectorizations} \label{heat_vectorization}
```

Comment:
Similarly to some other descriptions of representations, we do say what really is the vector representation ("the coordinate i,j is equal to... ") - we only say more or less what it corresponds to. 
For heat kernel, it's how it's done in the paper as well, i think (since you compute pairwise-evaluations).
For persistence images, we compute the integral of the smoothed out image. The authors define each pixel explicitly in [Def.2](https://arxiv.org/pdf/1507.06217.pdf).
While i am not saying that it is important to mention the distinction (especially since we do not call it persistence image anyway atm), just pointing out there is one.
What do you think?

Reply:
I think that if we write a persistence image entry we could explain this. How to go from vectorization into an infinite dimensional space into R^n. I.e. Persistence images are a composition of the symmetric vectorization and an average discreteization.